bitrise.yml

```diff
@@ -173,6 +173,10 @@ workflows:
             make clone-snapshots-ci repo_url="https://$GITHUB_USERNAME:$GITHUB_API_TOKEN@$SNAPSHOTS_REPO"
     - cache-pull@2: {}
     - certificate-and-profile-installer@1: {}
+    - script@1:
+        inputs:
+        - content: |-
+            bundle install
```

Comment:
Perhaps this change means that all workflows that build the app need to have a `bundle install` now, not only the one that runs =checks in the PRs.

Reply:
This commit will be dropped. Thank you for the notice